app/assets/stylesheets/modern-business.scss

```diff
@@ -161,7 +169,7 @@ header.carousel .fill#slider-3 {
 }
 
 .donation-candidate h3:hover {
-  color: #ff0000
+  color: #A30505;
```

Comment:
Color literals like `#A30505` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color `#A30505` should be written as `#a30505`